src/frontend/src/handler/create_source.rs

```diff
@@ -1465,6 +1466,14 @@ pub fn bind_connector_props(
                 .to_string(),
         );
     }
+    if !is_create_source && with_properties.is_mysql_cdc_connector() {
+        // Generate a random server id for mysql cdc source if needed
+        // `server.id` (in the range from 1 to 2^32 - 1). This value MUST be unique across whole replication
```

Comment:
How do we ensure uniqueness by generating a random number locally?

Reply:
We don't guarantee uniqueness, this is just a side note.

Reply:
> this is just a side note

I don't quite get this. So will this be a problem?